app/Mail/Membership/MembershipComplete.php

```diff
@@ -20,6 +20,11 @@ class MembershipComplete extends Mailable implements ShouldQueue
     */
     public $fullname;
 
+    /**
+     * @var string
+     */
+    public $membershipPIN;
```

Comment:
nitpick, but 
```suggestion
    public $membershipPin;
```

would also need same change in the view